### ISSUE TITLE
duplicate terms for ERAD

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12912]
GO:0030433
ER-associated ubiquitin-dependent protein catabolic process 
The chemical reactions and pathways resulting in the breakdown of proteins transported from the endoplasmic reticulum and targeted to cytoplasmic proteasomes for degradation. This process acts on misfolded proteins as well as in the regulated degradation of correctly folded proteins.

synonyms 
ER-associated protein breakdown, ER-associated protein catabolic process, ER-associated protein catabolism, ER-associated protein degradation, ERAD

and
GO:003650
ERAD pathway
The series of steps necessary to target endoplasmic reticulum (ER)-resident proteins for degradation by the cytoplasmic proteasome. Begins with recognition of the ER-resident protein, includes retrotranslocation (dislocation) of the protein from the ER to the cytosol, protein modifications necessary for correct substrate transfer (e.g. ubiquitination), transport of the protein to the proteasome, and ends with degradation of the protein by the cytoplasmic proteasome.

**Comment by ukemi** [comment]
It looks like the second term is more general. The first on is always ubiquitin-dependent. Are there other mechanisms?

**Comment by ValWood** [comment]
I think they are the same because the first on refers to the proteasome, and proteasomal degradation is ubiquitin dependent (or did we have an example where it isn't).
Anyways, I only know of one ERAD pathway.

If the wikipedia article is correct they are the same....
https://en.wikipedia.org/wiki/Endoplasmic-reticulum-associated_protein_degradation

**Comment by ukemi** [comment]
PMID:21969857 and PMID:20940304 ?

**Comment by ValWood** [comment]
Problem is, none of these has been used for ubiquitin independent by the looks of things....

**Comment by ValWood** [comment]
Ah got you. I though these were in different parts of the graph.

Could the names/defs be more consistent

ERAD pathway
and 
ubiquitin associated ERAD pathway

so that its easy to see the differentia based on term name?

**Comment by ValWood** [comment]
some of the direct children of 
GO:003650 ERAD pathway
should be under the ubiquitin dependent term


	
 GO:0000835 ER ubiquitin ligase complex
	
 GO:0000836 Hrd1p ubiquitin ligase complex
	
 GO:0000837 Doa10p ubiquitin ligase complex
	
 GO:0000838
Hrd1p ubiquitin ligase ERAD-M complex
	
 GO:0000839
Hrd1p ubiquitin ligase ERAD-L complex


GO:1904264
ubiquitin protein ligase activity involved in ERAD pathway


GO:0097466 glycoprotein ERAD pathway (as defined)

**Comment by ukemi** [comment]
Looking at the autophagy pathway structure, the catabolic process is a part of the whole pathway. Looking at the structure of the ERAD pathway, the catabolic process (GO:0030433) is a part of the pathway. Based on that and your comment above, I suspect that the catabolic term should only refer to the breakdown of the proteins and genes that function in other parts of the pathway should be annotated to such.

**Comment by ValWood** [comment]
I don't understand what the other parts of the pathway are? ErAD is a catabolic pathway?

**Comment by ukemi** [comment]
OK. We have decided to go the route of making these whole processes catabolic pathways. So based on the above, I can change the name of GO:0030433 to be ubiquitin-dependent ERAD pathway. I can tighten up the definition to be "The series of steps necessary to target endoplasmic reticulum (ER)-resident proteins for degradation by the cytoplasmic proteasome. Begins with recognition of the ER-resident protein, includes retrotranslocation (dislocation) of the protein from the ER to the cytosol, protein ubiquitination necessary for correct substrate transfer, transport of the protein to the proteasome, and ends with degradation of the protein by the cytoplasmic proteasome.

I will also move the children you suggest above to be children of the ubiquitin-dependent pathway and maybe change the name of the glycoprotein ERAD pathway to ubiquitin-dependent glycoprotein ERAD pathway. I think we should add the two PMIDs I found above to the definition of the parent term, so if this question ever comes up again, they will be easily available. Sound like a plan?
